src/updater.ts

```diff
@@ -137,14 +137,21 @@ class Updater {
   public loadDate(): void {
     this.lastUpdate = new Date(DataManager.getUpdaterData().lastUpdate);
   }
+  public updateHealthcheck(): void {
+    const data = DataManager.getUpdaterData();
+    data.healthcheckTimestamp = String(Date.now());
```

Comment:
It would be nice to save this in a human readable format, such as [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601), but we can change this later.

Reply:
fixed